src/requirements/data/majors/math.ts

```diff
@@ -0,0 +1,396 @@
+            'PHYS 4445',
+            'PHYS 4454',
+            'PHYS 4481',
+            'PHYS 4488',
+            'AEP 4340',
+            'AEP 4400',
+            'MATH 3***',
+            'MATH 4***',
+            'MATH 5***',
+            'MATH 6***',
+            'MATH 7***',
+          ]
+        ),
+        perSlotMinCount: [1, 3, 1],
+        slotNames: ['3000+ MATH Course', 'Physics Courses', 'MATH or Physics Course'],
+      },
+      'Operations Research': {
+        description:
+          '5 courses from (i) and (ii): ' +
+          '(i) At least one MATH course numbered 3000 or above, ' +
+          '(ii) At least three courses in ORIE in which the primary focus involves mathematical techniques.',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          [
+            'ORIE 3300',
+            'ORIE 3310',
+            'ORIE 3500',
+            'ORIE 3510',
+            'ORIE 4150',
+            'ORIE 4350',
+            'ORIE 4520',
+            'ORIE 4600',
+            'ORIE 4630',
+            'ORIE 4740',
+            'ORIE 5600',
+            'ORIE 5610',
+            'ORIE 5640',
+          ],
+          [
+            'ORIE 3300',
+            'ORIE 3310',
+            'ORIE 3500',
+            'ORIE 3510',
+            'ORIE 4150',
+            'ORIE 4350',
+            'ORIE 4520',
+            'ORIE 4600',
+            'ORIE 4630',
+            'ORIE 4740',
+            'ORIE 5600',
+            'ORIE 5610',
+            'ORIE 5640',
+            'MATH 3***',
+            'MATH 4***',
+            'MATH 5***',
+            'MATH 6***',
+            'MATH 7***',
+          ]
+        ),
+        perSlotMinCount: [1, 3, 1],
+        slotNames: ['3000+ MATH Course', 'ORIE Courses', 'MATH or ORIE Course'],
+      },
+      Statistics: {
+        description:
+          'MATH 4710, MATH 4720, ' +
+          'One additional MATH course numbered 3000 or above, ' +
+          'Two courses in other departments with significant content in statistics',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['MATH 4710'],
+          ['MATH 4720'],
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          [
+            'BTRY 4820',
+            'CS 4700',
+            'CS 4780',
+            'CS 4786',
+            'ECON 3140',
+            'ECON 4110',
+            'ORIE 4740',
+            'ORIE 4741',
+            'STSCI 3100',
+            'STSCI 3510',
+            'STSCI 4030',
+            'STSCI 4060',
+            'STSCI 4100',
+            'STSCI 4110',
+            'STSCI 4140',
+            'STSCI 4520',
+            'STSCI 4550',
+            'STSCI 4740',
+            'STSCI 4780',
+          ]
+        ),
+        perSlotMinCount: [1, 1, 1, 2],
+        slotNames: ['MATH 4710', 'MATH 4720', '3000+ MATH Course', 'Statistics Courses'],
```

Comment:
Instead of Statistics Courses, can you say Statistics Course?
![Screen Shot 2021-04-17 at 10 17 41 AM](https://user-images.githubusercontent.com/54298311/115116142-2586bd80-9f66-11eb-97bf-dd56efcfe785.png)